src/base/math/gcomplex.h

```diff
@@ -59,26 +63,26 @@ class GPUcomplex {
    * Utility constructor, creates class from given real and imaginary value
    */
   __host__ __device__ GPUcomplex(const floatT &real, const floatT &imag) {
-    c.x = real;
-    c.y = imag;
+    cREAL = real;
+    cIMAG = imag;
```

Comment:
I'm not sure if we should keep these defines (cREAL and cIMAG). I think from a design perspective I would drop them entirely and instead wrap floatT2 in a more C++ friendly way (e.g. a templated struct)

Reply:
these are also "artifacts" of some define replacements I did. I didn't bother to change them back, because there was no difference. For now I could just remove the defines and leave it as c.x and c.y everywhere?

Reply:
upon further inspection, these defines are used in a lot of other files as well. you could argue that it makes the code a bit easier to read from a physics viewpoint